webhook-connector/configure-the-webhook.md

```diff
@@ -4,50 +4,68 @@ description: Configurate the webhook in the Tradecloud portal
-* Select **Settings** in the page menu.
-* Select **Webhook** in the Integration section.
+* Select **My company** in the menu below your avatar.
+* The **Settings** tab in the Company menu should be selected.
+* Configure one or more **Webhook Integration**s below.
 
-{% hint style="info" %}
-The shipment webhook has a similar configuration section called "Shipments Webhook Integration".
+### Orders Webhook Integration
 
-The shipment webhook configuration will be moved into the integration section in the future.
-{% endhint %}
+Enable the **Order Webhook Integration** if you want to receive a webhook trigger when an order has been issued or changed:
+
+![Order Events](../.gitbook/assets/webhook-order-events.png)
+
+A default set of order events will be enabled. The actual set you want to select wil be dependent on the capabilities of your integration and ERP system. You can find a list of events here:
+
+{% page-ref page="order-events.md" %}
+
+![Order Delivery Schedule](../.gitbook/assets/webhook-order-delivery-schedule.png)
+
+In case of the order webhook you can choose between 
+* Multiple delivery lines per order line, also known as the **native** delivery schedule.
+* Only on delivery line per order line, also known as the **simple**  delivery schedule.
+
+See [the API manual](https://docs.tradecloud1.com/api/introduction/api/delivery-schedule) to read about the native versus the simple delivery schedule.
 
-### Select events to receive
+### Order Documents Webhook Integration
 
-A default set of events will be enabled. The actual set you want to receive wil be dependent on the capabilities of your integration and ERP system. You can find a list of events here:
+Enable the **Order Documents Webhook Integration** if you want to receive a webhook trigger when an order document has been attached:
 
-{% content-ref url="webhook-events.md" %}
-[webhook-events.md](webhook-events.md)
-{% endcontent-ref %}
+![Order Documents Events](../.gitbook/assets/webhook-order-documents-events.png)
 
-![](../.gitbook/assets/blue-sailboats-webhook-events.png)
+A default set of order document events will be enabled. The actual set you want to select wil be dependent on the capabilities of your integration and ERP system. You can find a list of events here:
+
+{% page-ref page="order-documents-events.md" %}
+
+### Shipments Webhook Integration
+
+Enable the **Order Webhook Integration** if you want to receive a webhook trigger when a shipment has been issued or changed:
+
+![Shipment Events](../.gitbook/assets/webhook-shipment-events.png)
+
+A default set of shipments events will be enabled. The actual set you want to select wil be dependent on the capabilities of your integration and ERP system. You can find a list of events here:
+
+{% page-ref page="shipment-events.md" %}
 
 ### Configure method, url and credentials
 
-![](../.gitbook/assets/blue-sailboats-webhook-url.png)
+![Order Webhook Congfig](../.gitbook/assets/webhook-order-config.png)
```

Comment:
Also render the other configuration panels side-by-side to avoid confusion? As it's merely an example..

Reply:
Not sure what you suggest.
The images are focussed and examples indeed.

Reply:
I meant using 3 images in 3 columns; for the order webhook settings, order document settings and shipment webhook settings. But it's a nit..

Reply:
I think its clear like this.